.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+    - "2.7"
+    - "3.4"
+
+install:
+  - pip install .
+
+services:
+  - docker
+
+before_install:
+  - docker pull enkeys/alpine-openjdk-amq7-snapshot
+  - docker run -d -p 5672:5672 enkeys/alpine-openjdk-amq7-snapshot amq7-server
+  - docker ps -a
+
+script:
+  - python tests/test_integration.py
```

Comment:
unfortunately, currently the tests must be executed from the _tests_ subdirectory, please change to that directory first.